packages/e2e-test-utils-playwright/src/editor/transform-block-to.ts

```diff
@@ -7,26 +7,38 @@ import type { Editor } from './index';
  * Clicks the default block appender.
  *
  * @param {Editor} this
- * @param {string} name Block name.
+ * @param {string} name Block name or title.
  */
 export async function transformBlockTo( this: Editor, name: string ) {
```

Comment:
Rewrites this to match
https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils/src/transform-block-to.js

Reply:
I should probably rewrite or remove the e2e-test-utils section in the [MIGRATION](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/MIGRATION.md#migration-steps-for-test-utils) guide. The utils are not meant to be one-to-one mapping and they don't necessarily have to behave the same. We try to match the names and the APIs because it's easier for the migration, but they should still follow the [best practices](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/README.md#best-practices). This is totally up for debate though as I mentioned in the [comment](https://github.com/WordPress/gutenberg/pull/45015#issuecomment-1300028659) 🙂.